waku/v2/node/waku_node.nim

```diff
@@ -258,10 +263,32 @@ proc new*(T: type WakuNode,
     peerManager: PeerManager.new(switch, peerStorage),
     switch: switch,
     rng: rng,
-    enr: enr,
+    enr: populatedEnr,
```

Comment:
This can be set to the `wakuDiscv5` enr if we have that available (in other words, move the construction to before node construction). We want to consolidate ENRs, so only need to create our own with `initEnr` if `wakuDiscv5` did not provide us with one.

Reply:
Addressed in [74c3622](https://github.com/waku-org/nwaku/pull/1512/commits/74c36223864222f0da14cee19f4a8f9b1f7f3654)